migrations/sql-migrations/README.md

```diff
@@ -33,3 +33,89 @@ MIGRATION_ENV=development npm run dynamoToSql
 ```
 
 If you want to migrate data for `MIGRATION_ENV` that is not equal to development, you will first need to run the `biomage rds tunnel` command (see more in [biomage-utils](https://github.com/hms-dbmi-cellenics/biomage-utils]).
+
+
+# Migrate users from one AWS account to another
+
+## Create users in target account
+To migrate users from one AWS account to another, you first need to create accounts for the users in the target account:
+
+```bash
+# AWS profile for target account
+export AWS_PROFILE=prod2
+biomage account create-users-list --user_list users_to_migrate.csv --input_env production
```

Comment:
```suggestion
```bash
# AWS profile for target account
export AWS_PROFILE=prod2
biomage account create-users-list --user_list users_to_migrate.csv --input_env development
```